receiver/sqlqueryreceiver/integration_test.go

```diff
@@ -36,305 +38,380 @@ const (
 	postgresqlPort = "5432"
 	oraclePort     = "1521"
 	mysqlPort      = "3306"
+	sqlServerPort  = "1433"
 )
 
-func TestPostgresIntegrationLogsTrackingWithoutStorage(t *testing.T) {
-	// Start Postgres container.
-	externalPort := "15430"
-	dbContainer := startPostgresDbContainer(t, externalPort)
-	defer func() {
-		require.NoError(t, dbContainer.Terminate(context.Background()))
-	}()
-
-	// Start the SQL Query receiver.
-	receiverCreateSettings := receivertest.NewNopSettings()
-	receiver, config, consumer := createTestLogsReceiverForPostgres(t, externalPort, receiverCreateSettings)
-	config.CollectionInterval = time.Second
-	config.Queries = []sqlquery.Query{
-		{
-			SQL: "select * from simple_logs where id > $1",
-			Logs: []sqlquery.LogsCfg{
-				{
-					BodyColumn:       "body",
-					AttributeColumns: []string{"attribute"},
-				},
+type DbEngine struct {
```

Comment:
I would inexpert that type just to be clear about it

Reply:
@atoulme Thank you for your comment. I'm not entirely clear on what you mean. Are you suggesting that I should simply rename this type to `DbEngineUnderTest`?